packages/api/src/Api.ts

```diff
@@ -12,16 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// import Types from '@cennznet/types/injects';
 import Types from '@cennznet/types/interfaces/injects';
 import { ApiPromise } from '@polkadot/api';
 import { ApiOptions as ApiOptionsBase, SubmittableExtrinsics } from '@polkadot/api/types';
 
-// import derives from './derives';
+import derives from './derives';
 import staticMetadata from './staticMetadata';
-// import { ApiOptions, Derives, SubmittableExtrinsics } from './types';
-import { ApiOptions } from './types';
-// import { mergeDeriveOptions } from './util/derives';
+// import { SubmittableExtrinsics } from './types';
```

Comment:
plan to remove SubmittableExtrinsic as part of clean up.. once we have all the test working..